samples/reproducible_build/Dockerfile

```diff
@@ -1,29 +1,41 @@
-FROM ubuntu:focal-20210609 AS build
+ARG egover=0.5.1
+FROM ghcr.io/edgelesssys/ego/build-base:v$egover AS build
+ARG egover
```

Comment:
I'm not sure if having the version as a variable actually makes this Dockerfile easier or if I should just put the version everywhere.

Reply:
Given that we properly want to update the default on each release anyway, I would just put the version in there for simplicity and let people edit it manually if want to use an older version. I don't think it's really needed to switch between versions with an argument for the sample, especially since it only works on fixed versions.

Reply:
no https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

Reply:
Yeah, just saw that myself and edited it. I think it's fine then.